meteor/lib/api/rest/interfaces.ts

```diff
@@ -0,0 +1,41 @@
+export interface RestAPIAdLibActions {}
+export interface RestAPIAdLibPieces {}
+export interface RestAPIBlueprints {}
+export interface RestAPIBucketAdlibActions {}
+export interface RestAPIBucketAdlibs {}
+export interface RestAPIBuckets {}
+export interface RestAPICoreSystem {}
+export interface RestAPIEvaluations {}
+export interface RestAPIExpectedMediaItems {}
```

Comment:
I think we should probably keep Expected Media Items out of the stable API, considering that it has been on a path to deprecation for a good while now.